flink-runtime/src/test/java/org/apache/flink/runtime/operators/CombineTaskTest.java

```diff
@@ -92,7 +92,35 @@ public void testCombineTask() {
 		
 		this.outList.clear();
 	}
-	
+
+	@Test
+	public void testOversizedRecordCombineTask() {
+		int keyCnt = 1;
+		int valCnt = 20;
+
+		addInput(new UniformRecordGenerator(keyCnt, valCnt, true));
+		addDriverComparator(this.comparator);
+		addDriverComparator(this.comparator);
+		setOutput(this.outList);
+
+		getTaskConfig().setDriverStrategy(DriverStrategy.SORTED_GROUP_COMBINE);
+		getTaskConfig().setRelativeMemoryDriver(combine_frac);
```

Comment:
I think you cannot reduce the memory further below to enforce records to be oversized. I think you need to change the data generator to provide you with an oversized record. I think the tests have a `UnionIterator` where you can mix an oversized record into the generating iterator.